index.html

```diff
@@ -940,8 +940,18 @@ <h2 class='informative'>Vary</h2>
       <section id="transmission-fixity" class="informative">
         <h2>Transmission Fixity</h2>
         <p>
-          Transmission fixity is verified by application of the <code>Digest</code> header
-          defined in [[RFC3230]] to <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+          Transmission fixity may be verified by including a <code>Digest</code> header (defined in
+          [[RFC3230]]) in <code>POST</code> and <code>PUT</code> requests for <a>LDP-NR</a>s.
+        </p>
+      </section>
+      <section id="persistence-fixity" class="informative">
+        <h2>Persistence Fixity</h2>
+        <p>
+          Persistence fixity may be verified by retrieving the current checksum of an <a>LDP-NR</a> and
```

Comment:
In any case, I tend to think that an explanation `Want-Digest` that spells out the inference is helpful in understanding what's going on, and how it can be used as a tool to support persistence fixity where applicable.    It also would take a bit of a softer stance on the questions of the nature of persistence.  i.e. if you have a system where you can't infer that `Want-Digest` can be used to support persistence fixity (maybe because the notion of persistence is complex) that's fine.